crates/ruff_linter/src/settings/mod.rs

```diff
@@ -38,13 +41,34 @@ pub mod flags;
 pub mod rule_table;
 pub mod types;
 
+#[macro_export]
+macro_rules! display_settings {
```

Comment:
I think some documentation would help to demystify the macro :) (what are the supported options)